localization/english/load_tips/load_tips_l_english.yml

```diff
@@ -97,9 +97,34 @@
  LOADING_TIP_95:0 "#T Did you know?#!\n\nMjordin Combatants in Icecrown commonly say, 'Hela take you!', which could refer to Odyn's daughter, Helya."
  LOADING_TIP_96:0 "“Beware the mist and fog, mortals; for with them comes your end.”\n #bold —Windan#!"
  LOADING_TIP_97:0 "“We mantid are an elder race. The pandaren you associate with - they are but children. They have their role to play. Each cycle, our young swarm their walls. The pandaren slay the weak. The strong return. With each generation, we grow ever stronger.”\n #bold —Kil'ruk the Wind-Reaver#!"
- LOADING_TIP_98:0 "#T Did you know?#!\n\nElder shaman have warned that wisps of Ragnaros, loyal servants of the Firelord, have taken to the Molten Front. Vigilance is advised."
+ LOADING_TIP_98:0 "#T Did you know?#!\n\nElder shamans have warned that wisps of Ragnaros, loyal servants of the Firelord, have taken to the Molten Front. Vigilance is advised."
  LOADING_TIP_99:1 "“I would die a thousand deaths to see you suffer!”\n #bold —King Ymiron#!"
  LOADING_TIP_100:0 "#T Did you know?#!\n\nAccording to Khadgar, Elisande's mastery of time magic rivaled that of the bronze dragonflight. This was due to the Nightwell's power."
+ LOADING_TIP_101:0 "“My son, the day you were born, the very forests of Lordaeron whispered the name Arthas. My child, I watched with pride as you grew into a weapon of righteousness. Remember, our line has always ruled with wisdom and strength. And I know you will show restraint when exercising your great power. But the truest victory, my son, is stirring the hearts of your people. I tell you this, for when my days have come to an end, you shall be King.”\n #bold —Terenas Menethil II, to his son, Arthas Menethil#!"
+ LOADING_TIP_102:0 "“Glad you could make it, Uther.”\n #bold —Arthas Menethil, to Uther the Lightbringer#!"
+ LOADING_TIP_103:0 "“Watch your tone with me, boy. You may be the prince, but I'm still your superior as a paladin!”\n #bold —Uther the Lightbringer, to Arthas Menethil#!"
+ LOADING_TIP_104:0 "“As if I could forget. Listen, Uther, there's something about the plague you should know...”\n #bold —Arthas Menethil, to Uther the Lightbringer#!"
+ LOADING_TIP_105:0 "“Oh, no. We're too late. These people have all been infected! They may look fine now, but it's just a matter of time before they turn into the undead!”\n #bold —Arthas Menethil, to Uther the Lightbringer#!"
+ LOADING_TIP_106:0 "“What?”\n #bold —Uther the Lightbringer, to Arthas Menethil#!"
```

Comment:
you literally put the whole dialogue in LMAOOOOO

Reply:
I got the urge to make an easter egg shitpost but if it's too wacky I can remove it

Reply:
nah this is funny as hell keep it